src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -28,7 +52,16 @@ def get_attrs(
         """
 
         # Preload to avoid fetching each user individually
-        prefetch_related_objects(item_list, "user", "inviter")
+        prefetch_related_objects(
+            item_list,
+            "user",
+            "inviter",
+            Prefetch(
```

Comment:
prefetch results in this type of query (used `self.assertNumQueries(0)` to get a read on the queries this madkes.

```sql
SELECT ("sentry_organizationmember_teams"."organizationmember_id") AS "_prefetch_related_val_organizationmember_id", "sentry_team"."id", "sentry_team"."organization_id", "sentry_team"."slug", "sentry_team"."name", "sentry_team"."status", "sentry_team"."actor_id", "sentry_team"."idp_provisioned", "sentry_team"."date_added", "sentry_team"."org_role" FROM "sentry_team" INNER JOIN "sentry_organizationmember_teams" ON ("sentry_team"."id" = "sentry_organizationmember_teams"."team_id") WHERE (NOT ("sentry_team"."org_role" IS NULL) AND "sentry_organizationmember_teams"."organizationmember_id" IN (1796))
```

Reply:
we can't use values in a Prefetch queryset and it appears `only` can't be used in Sentry at the moment haha https://github.com/getsentry/sentry/blob/2bbc0f425d9987dcf2062352a594155fdf1751a2/src/sentry/db/models/manager/base_query_set.py#L26

not sure if this will make a difference.